src/olympia/scanners/views.py

```diff
@@ -46,6 +46,13 @@ def get_queryset(self):
             bad_results = bad_results.filter(scanner=scanner)
             good_results = good_results.filter(scanner=scanner)
 
+        bad_filters = Q(state=TRUE_POSITIVE) | Q(
+            version__versionlog__activity_log__action__in=(
+                amo.LOG.BLOCKLIST_BLOCK_ADDED.id,
+                amo.LOG.BLOCKLIST_BLOCK_EDITED.id,
```

Comment:
I notice that there are two different reasons that a `good` result could be labelled `bad` as a result of being blocked (i.e., `BLOCKLIST_BLOCK_ADDED` and `BLOCKLIST_BLOCK_EDITED`), but it looks like a test was only added for one of these. Is that a concern, or do you feel the test cases are sufficient to cover the code change?

Reply:
I think it is fine as it is.

Reply:
Yes, we do generally take that approach, although I must say I'm not a huge fan of it. It does cut down on test code, but I actually think it's better if one can read a single test in isolation and be able to tell that it's correctly testing what needs to be tested. This approach, which, again, I know we commonly take, means that we have to have knowledge of other tests in the file to be sure that our test setup is correct.

Personally, I think one additional check assertion in a test adds more value than it takes away (due to maintenance), but I know this isn't how we usually do things, and this PR isn't necessarily the place to change that approach. In other words, let's leave it as is, but perhaps this is a topic worth discussing further.

Reply:
Oops, I added my reply to the wrong comment. I moved it.